logic/transaction.js

```diff
@@ -279,7 +279,8 @@ class Transaction {
 	/**
 	 * Returns true if a transaction was confirmed.
 	 *
-	 * @param {transaction} transaction
+	 * @param {Object} transaction
+	 * @param {string} transaction.id - only transaction id is necessary here
```

Comment:
There should be a single parameter.

Reply:
Don't want to interfere in the other code changes. Mind that I didn't change any single line of that function.

Reply:
But this code change comes from a commit of yours https://github.com/LiskHQ/lisk/pull/2512/commits/9826bf8c483cd473fcc19b3f7b3b8e8df7afd0b1